go/engine/pgp_encrypt.go

```diff
@@ -184,56 +183,22 @@ func (e *PGPEncrypt) loadSelfKey() (*libkb.PGPKeyBundle, error) {
 func (e *PGPEncrypt) verifyUsers(ctx *Context, assertions []string, loggedIn bool) ([]string, error) {
 	var names []string
 	for _, userAssert := range assertions {
-		var err error
-		var username string
-		if loggedIn && !e.arg.SkipTrack {
-			username, err = e.trackUser(ctx, userAssert)
-		} else {
-			username, err = e.identifyUser(ctx, userAssert)
+		arg := keybase1.Identify2Arg{
+			UserAssertion: userAssert,
+			Reason: keybase1.IdentifyReason{
+				Type: keybase1.IdentifyReasonType_ENCRYPT,
```

Comment:
To not short-circuit.  In this case, we'll get a weird U/I if we short-cirtcuit.